src/components/CMSForm/index.tsx

```diff
@@ -75,18 +75,17 @@ const RenderForm = ({ form }: { form: Form }) => {
           setIsLoading(false)
           setHasSubmitted(true)
 
-          if (confirmationType === 'redirect' && redirect) {
-            const { url } = redirect
+          if (confirmationType === 'redirect' && formRedirect) {
+            const { url } = formRedirect
 
             if (!url) return
 
             try {
-              const redirectUrl = new URL(url)
-
-              if (redirectUrl.origin === process.env.NEXT_PUBLIC_APP_URL) {
-                router.push(redirectUrl.href)
+              if (url.startsWith('/')) {
```

Comment:
I'd like to see the `URL` method still used here. You just need to work with the api a bit to get it to behave as you need. It accepts an additional `base` argument that you could like use. This is so that local urls written with full, the top-level domain will still be treated as a relative url.